pkgs/development/python-modules/pyobjc-core/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  buildPythonPackage,
+  darwin,
+  fetchFromGitHub,
+  lib,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pyobjc-core";
+  version = "11.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ronaldoussoren";
+    repo = "pyobjc";
+    tag = "v${version}";
+    hash = "sha256-RhB0Ht6vyDxYwDGS+A9HZL9ySIjWlhdB4S+gHxvQQBg=";
+  };
+
+  sourceRoot = "source/pyobjc-core";
+
+  build-system = [ setuptools ];
+
+  buildInputs = [
+    darwin.DarwinTools
+    darwin.libffi
+  ];
+
+  nativeBuildInputs = [
+    darwin.DarwinTools # sw_vers
+  ];
+
+  # See https://github.com/ronaldoussoren/pyobjc/pull/641. Unfortunately, we
+  # cannot just pull that diff with fetchpatch due to https://discourse.nixos.org/t/how-to-apply-patches-with-sourceroot/59727.
+  postPatch = ''
+    for file in Modules/objc/test/*.m; do
+      substituteInPlace "$file" --replace "[[clang::suppress]]" ""
+    done
```

Comment:
We could use grep here to avoid the deprecation warnings

Reply:
i'm not sure i understand. what are you proposing specifically?

Reply:
something like this:

```
for file in $(grep "[[clang::suppress]]" Modules/objc/test/*.m); do
```

Not sure if that is better though, just an idea